specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/iotSites.json

```diff
@@ -0,0 +1,261 @@
+            }
+          },
+          "201": {
+            "description": "Created IoT site",
+            "schema": {
+              "$ref": "#/definitions/IotSitesModel"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete IoT site": {
+            "$ref": "./examples/IotSites/Delete.json"
+          }
+        },
+        "tags": [
+          "IoT Sites"
+        ],
+        "operationId": "IotSites_Delete",
+        "description": "Delete IoT site",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/IotSiteScope"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Deleted IoT site"
+          },
+          "204": {
+            "description": "IoT site does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "IotSitesList": {
+      "description": "List of IoT sites",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List data",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IotSitesModel"
+          }
+        }
+      }
+    },
+    "IotSitesModel": {
+      "type": "object",
+      "description": "IoT site model",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "IoT site properties",
+          "$ref": "#/definitions/IotSiteProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "IotSiteProperties": {
+      "type": "object",
+      "description": "IoT site properties",
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "Display name of the IoT site"
+        },
+        "tags": {
+          "type": "object",
+          "description": "Tags of the IoT site",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      },
+      "required": [
+        "displayName",
+        "tags"
+      ]
+    }
+  },
+  "parameters": {
+    "IotSiteScope": {
+      "name": "scope",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Scope of the query (IoT Hub, /providers/Microsoft.Devices/iotHubs/myHub)",
```

Comment:
Do you support more than one resource type as a scope? If yes, should iotSites be modeled as a nested resource type instead of an extension resource type? (If you are planning to support more scopes in the future, the current modeling makes sense.)

Reply:
We support more scopes in the future

Reply:
outdated, this file already merged into master